src/Toml/Bi/Combinators.hs

```diff
@@ -293,6 +305,58 @@ arrayNonEmptyOf :: TomlBiMap a AnyValue -> Key -> TomlCodec (NonEmpty a)
 arrayNonEmptyOf = match . _NonEmpty
 {-# INLINE arrayNonEmptyOf #-}
 
+----------------------------------------------------------------------------
+-- Monoid codecs
+----------------------------------------------------------------------------
+
+{- | Codec for 'All' wrapper for boolean values.
+
+@since 1.2.1.0
+-}
+all :: Key -> TomlCodec All
+all = dimap (Just . getAll) (All . fromMaybe False) . dioptional . bool
```

Comment:
Maybe here should be `True` as an argument to `False`? Because `mempty` for `All` is `All True` but it's hard to guess for me what will be more useful :thinking:

Reply:
Ah, good catch! It could actually spoil smth.